src/__tests__/baseFields.int.test.ts

```diff
@@ -587,9 +587,8 @@ describe('/baseFields', () => {
 				})
 				.expect(200);
 			const after = await loadTableMetrics('base_field_localizations');
-			const baseFieldLocalizations = await loadBaseFieldLocalizationsBundle({
-				baseFieldId: 1,
-			});
+			const baseFieldLocalizations =
+				await loadBaseFieldLocalizationsBundleByBaseFieldId(1, 100, 0);
```

Comment:
Ah, this change highlights that we were leaning on the default parameters in the test!

To your point yesterday about not liking to pass around a bunch of bare numbers, would it be worth extracting some constants to use in the tests?

```suggestion
			const baseFieldLocalizations = await loadBaseFieldLocalizationsBundleByBaseFieldId(
			    testBaseFieldId,
			    testLimit,
			    testOffset,
			);
```

Reply:
Ok so (1) yes and (2) I just realized: limit should be able to be undefined (since passing undefined to limit is the same as saying "return everything"

I'm creating two constants for tests: `MAX_LIMIT` (undefined) and `MIN_OFFSET` (0).

For base field ID I'll just access the test base field object that is returned by `createTestBaseField()`